### PR TITLE
Extend for 1mom-microphysics granule

### PR DESCRIPTION
**Pull request by jonasjucker**
1mom-microphysics granule runs all schemes inside the same rundir to save runtime to create the perturbed input-data.
Therefore create folder for each member, not only once for all members.

Pipeline to check: https://gitlab.dkrz.de/icon/icon-nwp/-/pipelines/55443

Corresponding MR: https://gitlab.dkrz.de/icon/icon-nwp/-/merge_requests/1251

New tolerances look reasonable:
`/scratch/mch/jenkins/icon/pool/data/ICON/buildbot_data/ref/7e230cb75b6a022f4e3e57e553274a3e3b8c9147/mch_kenda-ch1_small_tolerance.csv`
or 
`/scratch/mch/jenkins/icon/pool/data/ICON/buildbot_data/ref/21a8c3389ed04e04ed96fb2569800189079dbd3f/mch_ch_lowres_tolerance.csv`

### CODE REVIEW

**Comment by jonasjucker** [comment]
launch jenkins

**Comment by jonasjucker** [comment]
@huppd Can you have a look at it?
We might need to introduce another config-option.

File: engine/run_ensemble.py [hunk 1]
```diff
@@ -73,16 +73,23 @@ def prepare_perturbed_run_script(
...
-                break
-
-        # rename the experiment name
-        if line == out_line:
-            out_line = replace_string(line, experiment_name, modified_experiment_name)
+        out_line = line
+        # only modify namelist if lhs,rhs_old or rhs_new are not equal None
+        if not all(
+            all(item is None for item in entry) for entry in [lhs, rhs_old, rhs_new]
+        ):
```
Comment (huppd) [inline engine/run_ensemble.py:80]: Maybe a matter of taste, but I would go with something like this:
```if any(entry is not None for entry in [lhs, rhs_old, rhs_new]):```
But I would definelty remove the if statement outside of the loop, it is constant for the whole loop.

**Review by huppd — COMMENTED** [review]
Except on small request, it looks good to me. I am a bit surprised that you need to generate new ICON references, that is a bit unexpected, isn't it?

**Comment by jonasjucker** [comment]
@huppd This should be ready to be merged.
The corresponding MR is about the be reviewed by Gatekeepers

**Pull request merged**